pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -235,6 +237,42 @@ func (e *cloudWatchExecutor) handleDescribeLogGroups(ctx context.Context,
 	return frame, nil
 }
 
+func (e *cloudWatchExecutor) handleDescribeAllLogGroups(ctx context.Context,
+	logsClient cloudwatchlogsiface.CloudWatchLogsAPI, parameters LogQueryJson) (*data.Frame, error) {
+
+	var namePrefix, nextToken *string
+	if len(parameters.LogGroupNamePrefix) != 0 {
+		namePrefix = aws.String(parameters.LogGroupNamePrefix)
+	}
+
+	var response *cloudwatchlogs.DescribeLogGroupsOutput
+	var err error
+	logGroupNames := []*string{}
+	for {
+		response, err = logsClient.DescribeLogGroupsWithContext(ctx, &cloudwatchlogs.DescribeLogGroupsInput{
+			LogGroupNamePrefix: namePrefix,
+			NextToken:          nextToken,
+			Limit:              aws.Int64(10),
```

Comment:
Any reason this is 10 and not 50 (default)?

Reply:
no, I was using that to test and forgot to commit the cleanup changes 😞